storage/azurestorage.py

```diff
@@ -232,41 +233,41 @@ def stream_upload_chunk(self, uuid, offset, length, in_fp, storage_metadata, con
         total_bytes_written = 0
 
         while True:
-            current_length = length - total_bytes_written
             max_length = (
-                min(current_length, self._max_block_size)
+                min(length - total_bytes_written, self._max_block_size)
```

Comment:
should `self._max_block_size` here also be set to the smaller block size (`1024 * 1024 * 128`)?

Reply:
Updated to use the minimum of that value or what the api supports.